frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java

```diff
@@ -57,6 +70,7 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpRequest req) {
 
             String[] segments = path.split("/");
             handleRequest(ctx, req, decoder, segments);
+            success = true;
```

Comment:
1. We don't if this is success or not
2. The response is send async.

We can collect metrics while response is sent.